hdf5.spec

```
@@ -13,18 +14,16 @@
 # You need to recompile all users of HDF5 for each version change
 Name: hdf5
 Version: %{major}.%{minor}
-Release: 4%{?dist}
+Release: 5%{hdf5_sha}%{?dist}
```

I'm noticing that the rpms are being built with a `55b5a1a8` release (`5` + the `5b5a1a8` sha):
```
[2020-11-06T15:33:46.189Z] Finish: rpmbuild hdf5-1.12.0-55b5a1a8.fc31.src.rpm
[2020-11-06T15:33:46.189Z] Finish: build phase for hdf5-1.12.0-55b5a1a8.fc31.src.rpm
```
Should a `-` or `.` be included here?
```suggestion
Release: 5-%{hdf5_sha}%{?dist}
```

---

%global hdf5_commit 5b5a1a81029eb7bdc975beff5f18c9c440f5de56
%define hdf5_sha .g5b5a1a8

Version: %{major}.%{minor}
Release: 5%{hdf5_sha}%{?dist}